packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -303,3 +268,181 @@ const localPreviewTrampoline = (mobileView: boolean, doNotShow?: boolean): JSX.E
   }
   return <LocalPreview mobileView={mobileView} showDevicesButton={mobileView} />;
 };
+
+const cameraPermissionGrantedTrampoline = (
+  cameraPermissionGranted: boolean | undefined,
+  videoState?: PermissionState | undefined
+): boolean | undefined => {
+  /* @conditional-compile-remove(call-readiness) */
+  return videoState ? videoState === 'granted' : cameraPermissionGranted;
+
+  return cameraPermissionGranted;
+};
+
+const micPermissionGrantedTrampoline = (
```

Comment:
nit: To be consistent, I would rename this as microphonePermissionGrantedTrampoline (though this will be omitted once this goes into stable)